css/style.css

```diff
@@ -1,3 +1,140 @@
 h1 {
-  color: salmon;
+  color: gray;
+  font-size: 50px;
+  font-family: sans-serif;
+}
+
+h2 {
+  color: gray;
+  font-size: 28px;
+  font-family: sans-serif;
+}
+
+body h1 {
+  text-align: center;
+  margin-top: 80px;
+  margin-left: 100px;
+  display: block;
+
+}
+
+body h2 {
+  text-align: center;
+  margin-top: 20px;
+  margin-left: 100px;
+
+}
+
+
+img {
+  text-align: center;
+  color: white;
+  font-family: sans-serif;
+  height: 400px;
+  width:300px;
+  float:left;
+  margin-right: 20px;
+  margin-left: 80px;
+
+}
+
+h3 {
+  color: White;
+  font-size: 50px;
+  font-family: sans-serif;
+  text-align: center;
+
+}
+
+#San-Francisco {
+  background: url('/Users/lgmusic24/css-airbnb/img/san-francisco.jpg');
```

Comment:
You'll want to use relative paths (`'../img/san-francisco.jpg'`) here. These paths will work on your machine but when someone else tries to clone the repo and open it, there isn't an image at this location